solidity-v1/dashboard/src/pages/threshold/HowItWorksPage.jsx

```diff
@@ -0,0 +1,136 @@
+        </NavLink>
+      </section>
+
+      <section className="tile threshold-how-it-works-page__resources">
+        <h3 className="mb-1">Upgrade Granted KEEP Tokens</h3>
+        <List>
+          <List.Title className="h5 text-violet-80">Released Tokens</List.Title>
+          <List.Content className="bullets bullets--violet-80 text-grey-60">
+            <List.Item className="mb-1">
+              Available released tokens need to be withdrawn to your wallet.
+              There are no partial withdrawals.
+            </List.Item>
+            <List.Item className="mb-1">
+              Liquid KEEP in your wallet can be upgraded to T.
+            </List.Item>
+            <List.Item className="mb-1">
+              The withdraw will be done in one transaction.
+            </List.Item>
+          </List.Content>
+        </List>
+
+        <List className="mt-3">
+          <List.Title className="h5 text-violet-80">
+            Withdrawn Tokens
+          </List.Title>
+          <List.Content className="bullets bullets--violet-80 text-grey-60">
+            <List.Item className="mb-1">
+              Withdrawn tokens are liquid KEEP in your wallet.
+            </List.Item>
+            <List.Item className="mb-1">
+              Migrate your liquid KEEP to T in the portal.
+            </List.Item>
+            <List.Item className="mb-1">
+              You can upgrade whatever amount of tokens you want in one single
+              transaction.
+            </List.Item>
+          </List.Content>
+        </List>
+
+        <List className="mt-3">
+          <List.Title className="h5 text-violet-80">
+            Staked Granted Tokens
+          </List.Title>
+          <List.Content className="bullets bullets--violet-80 text-grey-60">
+            <List.Item className="mb-1">
+              If you are staking a part of your granted tokens you can either
+              opt to undelegate your tokens or continue staking on Keep Network.
+            </List.Item>
+            <List.Item className="mb-1">
+              If you undelegate, there is a 60 day undelegation period.
+            </List.Item>
+            <List.Item className="mb-1">
+              After 60 days, you can withdraw your tokens and upgrade your KEEP
+              to T if they are available to release from your token grant.
+            </List.Item>
+          </List.Content>
+        </List>
+      </section>
+
+      <section className="tile threshold-how-it-works-page__upgrade-liquid-keep">
+        <h3 className="mb-1">Upgrade Liquid KEEP Tokens</h3>
+        <p className="text-grey-60">
+          You can upgrade any amount of your liquid KEEP tokens in one
+          transaction on the Threshold dapp.
+        </p>
+        <div className="threshold-how-it-works-page__upgrade-liquid-keep__button-container">
+          <Icons.TTokenSymbol />
+          <h4 className="button-container__title">Threshold</h4>
+          <a
+            href={LINK.tbtcDapp}
```

Comment:
Can we add the new property to the `LINK` variable? eg

```
{
 ...
 thresholdDapp: "https://dashboard.test.threshold.network/"
}
```

Reply:
Fixed in d98495c2b3bb7b2a92f266850e03be3c46150270